javalib/src/main/scala/java/math/RoundingMode.scala

```diff
@@ -25,12 +25,8 @@ package java.math
 
 import scala.annotation.switch
 
-final class RoundingMode private (_index: Int, _name: String) {
-  def name(): String = _name
-  def ordinal(): Int = _index
-
-  override def toString(): String = name()
-}
+final class RoundingMode private (_index: Int, _name: String)
+    extends Enum[RoundingMode](_name, _index)
```

Comment:
The Boy Scout Rule suggests that the order of the parameters of `RoundingMode` be reversed to match that of `Enum`.
The same applies to `TimeUnit` and `sbt.testing.Status`.